docker/docker-compose.yml.dist

```diff
@@ -0,0 +1,73 @@
+version: "3.5"
+
+services:
+
+  php:
+    build: docker/php
+    container_name: php.uitdatabank
+    volumes:
+      - .:/var/www/html:cached
+    working_dir: /var/www/html/web
+    command: php -S 0.0.0.0:8000
+    ports:
+      - "8000:8000"
+    networks:
+      uitdatabank:
+        aliases:
+          - php.uitdatabank.dev
+
+  mysql:
+    image: 'mysql/mysql-server:8.0'
+    container_name: mysql.uitdatabank
+    environment:
+      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
+      - MYSQL_DATABASE=${MYSQL_DATABASE}
+      - MYSQL_USER=${MYSQL_USER}
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+    volumes:
+      - 'udb3-mysql:/var/lib/mysql'
+    networks:
+      uitdatabank:
+        aliases:
+          - mysql.uitdatabank.dev
+    ports:
+      - "3306:3306"
+
+  rabbitmq:
```

Comment:
Maybe we need to remove `rabbitmq` because it's not used on M1, or should we try it on another Mac?